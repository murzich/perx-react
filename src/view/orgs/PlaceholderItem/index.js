import PlaceholderItem from './PlaceholderItem';

export default PlaceholderItem;
