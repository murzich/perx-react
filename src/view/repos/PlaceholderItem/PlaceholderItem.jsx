import React from 'react';
import { Placeholder } from 'semantic-ui-react';

const PlaceholderItem = () => (
  <Placeholder>
    <Placeholder.Header>
      <Placeholder.Line length="short" />
    </Placeholder.Header>
    <Placeholder.Paragraph>
      <Placeholder.Line length="full" />
      <Placeholder.Line length="short" />
    </Placeholder.Paragraph>
    <Placeholder.Paragraph>
      <Placeholder.Line length="medium" />
    </Placeholder.Paragraph>
  </Placeholder>
);

export default PlaceholderItem;
