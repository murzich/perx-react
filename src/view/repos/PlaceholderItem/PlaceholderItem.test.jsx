import { cleanup, render } from '@testing-library/react';
import React from 'react';
import PlaceholderItem from './PlaceholderItem';

describe('PlaceholderItem', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { asFragment } = render(<PlaceholderItem />);

    expect(asFragment()).toMatchSnapshot();
  });
});
