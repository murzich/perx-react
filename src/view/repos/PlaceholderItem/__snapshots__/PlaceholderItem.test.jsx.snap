// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceholderItem should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="ui placeholder"
  >
    <div
      class="header"
    >
      <div
        class="line short"
      />
    </div>
    <div
      class="paragraph"
    >
      <div
        class="line full"
      />
      <div
        class="line short"
      />
    </div>
    <div
      class="paragraph"
    >
      <div
        class="line medium"
      />
    </div>
  </div>
</DocumentFragment>
`;
